install/installer/scripts/kots-install.sh

```diff
@@ -156,6 +156,9 @@ EOF
         HELM_TIMEOUT="1h"
     fi
 
+    echo "Gitpod: shut down any running workspaces/image-builders"
+    kubectl delete pods -n "${NAMESPACE}" -l component=workspace --wait
```

Comment:
So this would mean that during upgrade the workspaces that are currently in use will get interrupted? I thought it was a nice feature that we had, allowing uninterrupted upgrades. And how would this affect potential backup jobs?

Reply:
As I understand it, this has been requested by @gitpod-io/engineering-workspace to guarantee that the workspaces are properly backed up. If you check out #13147, there's a whole discussion on this in there

Reply:
totally missed the context :see_no_evil: reading!